pepper-apis/parent-pom.xml

```diff
@@ -145,7 +151,12 @@
         <dependency>
             <groupId>mysql</groupId>
             <artifactId>mysql-connector-java</artifactId>
-            <version>5.1.42</version>
+            <version>8.0.18</version>
```

Comment:
Google Cloud mysql databases are at v5.7, is a v8 connector compatible with it? In either case, I think we might need to bump this version up (to at least 5.1.47?) in order to play nice with `utf8mb4`, as described [here](https://dev.mysql.com/doc/connector-j/5.1/en/connector-j-reference-charsets.html).

Reply:
I think we are good with the newer Java driver:
https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-versions.html